Tactics/CSE.lean

```diff
@@ -104,6 +104,13 @@ structure CSEConfig where
   minOccsToCSE : Nat := 2
   /-- Whether the tactic should throw an error if no CSEable subterms were found. -/
   failIfUnchanged : Bool := true
+  /-- Number of steps the tactic should spend searching for subterms to gather information. -/
+  fuelSearch : Nat := 1000
+  /-- Number of steps the tactic should spend performing subexpression elimination. -/
+  fuelEliminate : Nat := 1000
+  /-- Whether we are performing a dry run, where we will identify repeated
+      subterms, and simulate the sharing algorithm, but perform no actual sharing. -/
+  dryRun? : Bool := false
```

Comment:
Minor not: I tend to use `?` as a suffix to indicate something is an `Option`, so I'd prefer calling this just `dryRun`, or `isDryRun` if you want to emphasize it's a Bool

Reply:
Changed accordingly.